### PR TITLE
Update api-update-incidents.md

### PR DESCRIPTION
**Pull request by anz999**
Updated update incident API with possible value InProgress.

### CODE REVIEW

File: microsoft-365/security/defender/api-update-incidents.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ In the request body, supply the values for the fields that should be updated. Ex
 
 Property|Type|Description
 ---|---|---
-status|Enum|Specifies the current status of the incident. Possible values are: `Active`, `Resolved`, and `Redirected`.
+status|Enum|Specifies the current status of the incident. Possible values are: `Active`, `Resolved`,`InProgress` and `Redirected`.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/api-update-incidents.md]:
```suggestion
status|Enum|Specifies the current status of the incident. Possible values are: `Active`, `Resolved`, `InProgress`, and `Redirected`.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @anz999, kindly commit the minor suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@anz999 Gentle reminder. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR.Thanks

**Pull request merged**